newIDE/app/src/EventsSheet/InstructionEditor/NewInstructionEditorDialog.js

```diff
@@ -146,6 +171,30 @@ export default function NewInstructionEditorDialog({
     }
   };
 
+  const _addBehavior = (type: string, defaultName: string) => {
+    if (!chosenObject) return;
+    setNewBehaviorDialogOpen(false);
```

Comment:
should the closing of the dialog happen after the behavior is added? In case it fails or if there is a confirmation dialog happening?